Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2894,8 +2898,8 @@ extension BrowserViewController: HomeMenuControllerDelegate {
 
 extension BrowserViewController: TopSitesDelegate {
     
-    func didSelectUrl(url: URL) {
-        finishEditingAndSubmit(url, visitType: .bookmark)
+    func didSelect(urlString: String) {
```

Comment:
This isn't technically a `url` anymore, right? Maybe maybe something like `input` instead?